apps/sms/views/conversation/test/unit/link_helper_test.js

```diff
@@ -153,12 +153,14 @@ suite('link_helper_test.js', function() {
       test('non ascii prefix', function() {
         testURLMatch('àmozilla.org', 'mozilla.org', true);
       });
+      test('Bug 941763', function() {
```

Comment:
@kumarrishav , I am not sure. But I think it's just means of including reference to more details to the particular test. A new contributor can go to the bug page and find more details. But I know it's not consistent! :P